source/options.css

```diff
@@ -1,5 +1,6 @@
 :root {
 	--red-by-github: #cb2431;
+	min-width: 400px;
```

Comment:
Needed because Chrome loads the options lightbox at `width:0` before letting it fit the content. Fit-textarea needs to run on the correct size or else the content will be detected as infinitely long